types/contest.ts

```diff
@@ -1,7 +1,7 @@
 import { DateString, RelativeURL } from "./shared";
 
 // all values are strings because the source is a CSV file -- changed getting values from api instread of contest.csv
-export interface Contest {
+export interface DBContest {
```

Comment:
won't this break imports from other files, like manage-findings\.ts and others?

<img width="462" alt="search in VSCode for imports from types/contest showing other uses of older Contest name" src="https://user-images.githubusercontent.com/265902/233155541-a663af73-9dcb-4781-a4ee-c30e686386ea.png">



unless I'm missing something lets keep this as:

```suggestion
export interface Contest {
```

Reply:
No I left the contest as an interface, Contest extends from the DBContest to add what is missing and I didn't change any of the types in the interface.

Reply:
sorry missed that you sitll have `export interface Contest` below.